CMakeLists.txt

```diff
@@ -0,0 +1,2075 @@
+target_link_libraries(mixxx PUBLIC mixxx-lib)
+
+#
+# Installation and Packaging
+#
+include(GNUInstallDirs)
+install(
+  TARGETS mixxx
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+)
+
+# Skins
+install(
+  DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/res/skins
+  DESTINATION ${CMAKE_INSTALL_DATADIR}/mixxx
+)
+
+# Controller mappings
+install(
+  DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/res/controllers
+  DESTINATION ${CMAKE_INSTALL_DATADIR}/mixxx
+)
+
+# Translation files
+install(
+  DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/res/translations
+  DESTINATION ${CMAKE_INSTALL_DATADIR}/mixxx
+  FILES_MATCHING PATTERN "*.qm"
+)
+# Font files
+install(
+  DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/res/fonts
+  DESTINATION ${CMAKE_INSTALL_DATADIR}/mixxx
+)
+
+# Keyboard mapping(s)
+install(
+  DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/res/keyboard
+  DESTINATION ${CMAKE_INSTALL_DATADIR}/mixxx
+)
+
+# Documentation
+install(
+  FILES
+    ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE
+    ${CMAKE_CURRENT_SOURCE_DIR}/README
+    ${CMAKE_CURRENT_SOURCE_DIR}/Mixxx-Manual.pdf
+  DESTINATION ${CMAKE_INSTALL_DOCDIR}/mixxx
+)
+
+# .desktop file for KDE/GNOME menu
+install(
+  FILES ${CMAKE_CURRENT_SOURCE_DIR}/res/linux/mixxx.desktop
+  DESTINATION ${CMAKE_INSTALL_DATADIR}/applications/mixxx
+)
+
+# Icon file for menu entry
+install(
+  FILES ${CMAKE_CURRENT_SOURCE_DIR}/res/images/mixxx_icon.svg
+  DESTINATION ${CMAKE_INSTALL_DATADIR}/pixmaps/rules.d
+)
+
+# .appdata.xml file for KDE/GNOME AppStream initiative
+install(
+  FILES ${CMAKE_CURRENT_SOURCE_DIR}/res/linux/mixxx.appdata.xml
+  DESTINATION ${CMAKE_INSTALL_DATADIR}/appdata
+)
+
+# udev rule file for USB HID and Bulk controllers
+install(
+  FILES ${CMAKE_CURRENT_SOURCE_DIR}/res/linux/mixxx.usb.rules
+  DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/udev/rules.d
+)
+
+# Packaging
+set(CPACK_PACKAGE_VENDOR "Mixxx Project")
+set(CPACK_PACKAGE_CONTACT "RJ Skerry-Ryan <rryan@mixxx.org>")
+set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/cmake/cpack_package_description.txt")
+set(CPACK_DEBIAN_PACKAGE_SECTION "sound")
+set(CPACK_DEBIAN_PACKAGE_PRIORITY "optional")
+set(CPACK_DEBIAN_PACKAGE_SUGGESTS "pdf-viewer")
+set(CPACK_DEBIAN_PACKAGE_REPLACES "mixxx-data")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "libqt5opengl5, libqt5svg5, libqt5xml5, libqt5sql5, libqt5sql5-sqlite")
+set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS ON)
+include(CPack)
+
+#
+# Dependencies
+#
+option(STATIC_DEPS "Link dependencies statically" OFF)
+
+# Chromaprint
+find_package(Chromaprint REQUIRED)
+target_link_libraries(mixxx-lib PUBLIC Chromaprint::Chromaprint)
+if(WIN32 AND STATIC_DEPS)
+  target_compile_definitions(mixxx-lib PUBLIC CHROMAPRINT_NODLL)
+  find_package(FFTW REQUIRED)
+  target_link_libraries(mixxx-lib PUBLIC FFTW::FFTW)
+endif()
+
+# Ebur128
+find_package(Ebur128)
+cmake_dependent_option(EBUR128_STATIC "Link libebur128 statically" OFF "Ebur128_FOUND" ON)
+if(EBUR128_STATIC)
+  set(EBUR128_CMAKE_ARGS "-DBUILD_STATIC_LIBS=ON")
```

Comment:
A message: 
Looking for internal libebur128 
would be nice here.

Reply:
Done.